proposal/package-importer.d.ts.md

```diff
@@ -256,6 +256,8 @@ Javascript Compile API, insert:
 * If any object in `options.importers` is exactly equal to the object
   `nodePackageImporter`:
 
+  * If the execution environment is the browser, throw an error.
```

Comment:
@nex3 Is a more formal way needed to specify the execution environment?

Reply:
I think this is fine, although it might be slightly more accurate to say "if no filesystem is available" in case this is run in a different kind of sandboxed JS environment.

Reply:
Updated in https://github.com/sass/sass/pull/3699/commits/85a27d87d6cc5fc29ce878592a7edfb3a9042e35